packages/docs/src/components/layout/SideNav/SideNav.tsx

```diff
@@ -132,7 +134,10 @@ const SideNav = ({ location }: SideNavProps) => {
         // hidden attr applied on mobile breakpoints when nav is closed
         hidden={isMobile && !isMobileNavOpen}
       >
-        <div className="c-navigation__switchers-wrapper ds-u-display--none ds-u-md-display--block">
+        <div
+          className="c-navigation__switchers-wrapper ds-u-display--none ds-u-md-display--block"
+          style={{ backgroundColor: getThemeColorValue(theme, 'primary') }}
```

Comment:
This does work, I promise. If you aren't seeing the correct background color reflected in the theme switcher, navigate to another page on the site and it should "wake up." This is related to a bug I mentioned in #2922